src/Analyzers/Core/CodeFixes/ConflictMarkerResolution/AbstractConflictMarkerCodeFixProvider.cs

```diff
@@ -280,6 +280,29 @@ private static bool IsConflictMarker(TextLine currentLine, char ch)
             var secondMiddlePos = secondMiddleLine.Start;
             var endPos = endLine.Start;
 
+#if !CODE_STYLE
+
+            context.RegisterCodeFix(
+                CodeAction.DocumentChangeAction.Create(takeTopText,
+                    c => TakeTopAsync(document, startPos, firstMiddlePos, secondMiddlePos, endPos, c),
+                    TakeTopEquivalenceKey,
+                    CodeActionPriority.High),
+                context.Diagnostics);
+            context.RegisterCodeFix(
+                CodeAction.DocumentChangeAction.Create(takeBottomText,
+                    c => TakeBottomAsync(document, startPos, firstMiddlePos, secondMiddlePos, endPos, c),
+                    TakeBottomEquivalenceKey,
+                    CodeActionPriority.High),
+                context.Diagnostics);
+            context.RegisterCodeFix(
+                CodeAction.DocumentChangeAction.Create(CodeFixesResources.Take_both,
+                    c => TakeBothAsync(document, startPos, firstMiddlePos, secondMiddlePos, endPos, c),
+                    TakeBothEquivalenceKey,
+                    CodeActionPriority.High),
+                context.Diagnostics);
```

Comment:
this was the only high-pri provider not returning a high-pri code action.  Fixed that here.